other/edge-server/backend/data_sources/mongodb-atlas/todo/Item/rules.json

```
@@ -3,15 +3,15 @@
     "database": "todo",
     "roles": [
         {
-            "name": "readAndWriteAll",
+            "name": "edgeServerReadWriteAll",
```

If we're going to update this so the Edge Server can only write to its "own" documents, we should probably rename the rule.
```suggestion
            "name": "edgeServerReadAllWriteOwn",
```

---

Yeah, that's sensible. 😆